mapillary/models/api/vector_tiles.py

```diff
@@ -303,12 +449,16 @@ def __zoom_range_check(self, layer: str, zoom: int):
                 # Parameters accordingly
                 param="layer",
                 value=layer,
-                options=["overview", "sequence", "image"],
+                options=[
+                    "overview",
+                    "sequence",
+                    "image",
+                    "map_feature",
```

Comment:
not sure if this corresponds somewhere else to a variable, but the map_feature endpoint will look for layer name "point" and the traffic_sign endpoint will look for layer name "traffic_sign" -- see here: https://www.mapillary.com/developer/api-documentation/#traffic-sign-tiles

Reply:
Yep. I must've slipped. I did create logic for the two layers here. Fixed